From 343f00608d159217169e003d231b5e96a47d81f0 Mon Sep 17 00:00:00 2001
From: Binyuan Lan <lby@rock-chips.com>
Date: Sun, 6 Jan 2019 19:53:34 +0800
Subject: [PATCH] ASoC: rockchip: rk817-codec: solve sound L/R channel
 conversion problem

There is playback's sound L/R channel conversion problem when recording is opened

Change-Id: Iae7160e25bdd834df9904fbd34fe964449c93560
Signed-off-by: Binyuan Lan <lby@rock-chips.com>
---
 sound/soc/codecs/rk817_codec.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/codecs/rk817_codec.c b/sound/soc/codecs/rk817_codec.c
index ee4485135681..3b819a082c4d 100644
--- a/sound/soc/codecs/rk817_codec.c
+++ b/sound/soc/codecs/rk817_codec.c
@@ -316,7 +316,7 @@ static struct rk817_reg_val_typ capture_power_up_list[] = {
 	{RK817_CODEC_APLL_CFG4, 0xa5},
 	{RK817_CODEC_APLL_CFG5, 0x00},
 
-	{RK817_CODEC_DI2S_RXCMD_TSD, 0x00},
+	/*{RK817_CODEC_DI2S_RXCMD_TSD, 0x00},*/
 	{RK817_CODEC_DI2S_RSD, 0x00},
 	/* {RK817_CODEC_DI2S_CKM, 0x00}, */
 	{RK817_CODEC_DI2S_RXCR1, 0x00},
-- 
2.35.3

